python/wpt/run.py

```diff
@@ -73,6 +73,8 @@ def run_tests(**kwargs):
     set_if_none(
         kwargs, "host_cert_path", os.path.join(CERTS_PATH, "web-platform.test.pem")
     )
+    # set id_hash as default chunker
```

Comment:
I think we should leave a more detailed comment here so we remember what this is useful for.

Here's a proposal:

```
# Set `id_hash` as the default chunk, as this better distributes testing across different 
# chunks and leads to more consistent timing on GitHub Actions.
```